Core/include/Acts/Vertexing/AdaptiveGridTrackDensity.ipp

```diff
@@ -78,7 +78,7 @@ template <int trkGridSize>
 typename Acts::AdaptiveGridTrackDensity<trkGridSize>::DensityMap
 Acts::AdaptiveGridTrackDensity<trkGridSize>::addTrack(
     const Acts::BoundTrackParameters& trk, DensityMap& mainDensityMap) const {
-  SquareMatrix2 cov = trk.covariance().value().block<2, 2>(0, 0);
+  SquareMatrix2 cov = trk.impactParameterCovariance().value();
```

Comment:
Here, the default of `2` and hence $d_0, z_0$ would not be obvious to me, for example.